_posts/2018-09-06-how-we-use-pivotal-tracker-at-ombu-labs.markdown

```diff
@@ -2,155 +2,121 @@
+- Did I test many cases and it failed with an edge case?
 
 <img src="/blog/assets/images/user-story-rejecting-story.png" alt="State of a user story after you deliver it" class="medium-img">
 
-If possible, describe the exact scenario and steps that you took to find a
-problem with the user story. If you think that adding a screenshot will help,
-you should add it as a comment after you reject the story.
+If possible, a description of the exact scenario and steps that were taken to find a problem with the user story should be provided. If adding a screenshot or a video would help, one should be added as a comment before the story is rejected.
 
 ## Step 7: Restarting a rejected user story
 
-There will be times when your implementation is not good enough. It may fall
-short in terms of usability, performance, or behavior. The good news is that
-our process works. Someone verified a feature you implemented and found issues
-with the implementation.
+There will be times when an implementation is not good enough. It may fall short in terms of usability, performance, or behavior. The good news is that our process works. Someone verified a feature you implemented and found issues with the implementation.
 
 When the time comes, you will find a user story in this state:
 
 <img src="/blog/assets/images/user-story-restart.png" alt="State of a user story after someone rejects it" class="medium-img">
 
-You can simply restart the user story and resume working on it. If you need to
-submit another pull request to patch the feature, you can continue with step 4.
+You can simply restart the user story and resume working on it. If you need to submit another pull request to patch the feature, you can continue with step 4.
+
+## What if there are issues after merging?
+
+We should not be shipping bugs to production. However, if after the story has been accepted and deployed to production an issue is found, the story **should not be changed to rejected**.
+
+At that point, a **new** user story should be created with the type set to **bug** and the issue found should be described. The team will then proceed to follow the process to fix the bug.
 
 ## Conclusion
 
-[Pivotal Tracker](https://www.pivotaltracker.com/) provides a great way to track team velocity. This helps in determining when a big feature will be shipped. As much as we like
-to use [GitHub](https://github.com/) for everything, Pivotal Tracker integrates nicely with them (they recently launched this integration!) which improves traceability from feature to
-implementation.
+[Pivotal Tracker](https://www.pivotaltracker.com/) provides a great way to track project progress and team velocity. This helps in determining when a big feature will be shipped. As much as we like to use [GitHub](https://github.com/) for everything, Pivotal Tracker integrates nicely with them (they recently launched this integration!) which improves traceability from feature to implementation.
```

Comment:
Maybe we can link the help article on github integration here https://www.pivotaltracker.com/help/articles/github_integration/

Reply:
And maybe we can also say that it integrates nicely with Slack.